arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1550,7 +1575,7 @@
       "x-ms-azure-resource": true,
       "description": "The Resource definition."
```

Comment:
Yes Anu, Actually I got bit confused. your right. will make these changes and update the PR